Sources/Features/Breadcrumb/BacktraceBreadcrumbFileHelper.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+import Cassette
+
+enum BacktraceBreadcrumbFileHelperError: Error {
+    case invalidFormat
+}
+
+@objc public class BacktraceBreadcrumbFileHelper: NSObject {
```

Comment:
What do you think about renaming class to something like backtraceBreadcrumbFileManager? The class doesn't look like a helper class because we have a lot of business logic here that doesn't look like a typical helper.

Reply:
Good point. In this case it's the mirror image of the same "helper" in android. So I'll keep it the same for now, but record a ticket to update both. Fair?